packages/engine/vscode/src/lib/filemanager.ts

```diff
@@ -1,49 +1,62 @@
 import { filSystemProfile, IFileSystem } from '@remixproject/plugin-api'
 import { MethodApi } from '@remixproject/plugin-utils';
+import { isAbsolute, join } from 'path';
 import { window, workspace, Uri, commands } from 'vscode';
 import { CommandPlugin } from './command';
 
+function absolutePath(path: string) {
+  const root = workspace.workspaceFolders[0].uri.fsPath;
+  if (isAbsolute(path) || !root) {
+    return path;
+  }
+  return join(root, path);
+}
+
 export class FileManagerPlugin extends CommandPlugin implements MethodApi<IFileSystem> {
   constructor() {
     super(filSystemProfile);
   }
   /** Open the content of the file in the context (eg: Editor) */
   async open(path: string): Promise<void> {
-    const uri = Uri.parse(path);
+    const absPath = absolutePath(path);
```

Comment:
you don't need to parse it anymore?

Reply:
or is `Uri.file(absPath)` doing that?

Reply:
parse wasn't working as expected. Uri.file does (even for directories).